### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by usmangt**
The troubleshooting page was missing how to send dashboard panels for debugging so added it here.

**What is this feature?**

[Add a brief description of what the feature or update does.]

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer**:

I added the information on how to Troubleshoot Dashboard panels for debugging. This is v.frequely asked by the Engineers once the user reports an issue on GitHub and as there is no documentation exist how to do it but the feature is already available in Grafana.

Therefore I am making this PR request. (This is my 1st PR so incase if there are any mistakes then kindly let me know :slightly_smiling_face:  )

### CODE REVIEW

**Comment by chri2547** [comment]
Hi @usmangt , I believe the content you want to contribute is already available at: https://grafana.com/docs/grafana/latest/troubleshooting/send-panel-to-grafana-support/. If that document meets your needs, I think we can close this PR. Thank you.

**Comment by usmangt** [comment]
Hi @chri2547 ,

Thanks for the review. Yes, this is exactly what I wanted to do (which is already there in a better way with screenshots).

I kind of missed it because the Table of content did not show it inside the main Troubleshooting page as it is listed as a sub-page.

We can close this PR.

**Comment by jdbaldry** [comment]
> I kind of missed it because the Table of content did not show it inside the main Troubleshooting page as it is listed as a sub-page.

@chri2547 We could consider adding a `section` shortcode here?

I'll leave it up to you :)

File: docs/sources/troubleshooting/_index.md [hunk 1]
```diff
@@ -23,6 +23,16 @@ You can enable more logging by changing log level in the Grafana configuration f
 
 For more information, refer to [Enable debug logging in Grafana CLI]({{< relref "../cli/#enable-debug-logging" >}}) and the [log section in Configuration]({{< relref "../setup-grafana/configure-grafana/#log" >}}).
 
+## Troubleshoot with Dashboards Panels
+
+If you have an issue with your Dashboard panel, then simply send us the debug information using the following steps.
+
+Open the dropdown menu by clicking on your panel title -> Hover mouse to **More** -> Then select **Get Help**
+
+Then it will open a new page and ask you to send the snapshot data. You can simply click the button **Dashobard** to Download it and then send it for debugging. Also if you have any sensitive data then you can use the **Randomize data** option which will hide the original data.
```
Comment (chri2547) [inline docs/sources/troubleshooting/_index.md:32]:
```suggestion
```

File: docs/sources/troubleshooting/_index.md [hunk 2]
```diff
@@ -23,6 +23,16 @@ You can enable more logging by changing log level in the Grafana configuration f
 
 For more information, refer to [Enable debug logging in Grafana CLI]({{< relref "../cli/#enable-debug-logging" >}}) and the [log section in Configuration]({{< relref "../setup-grafana/configure-grafana/#log" >}}).
 
+## Troubleshoot with Dashboards Panels
+
+If you have an issue with your Dashboard panel, then simply send us the debug information using the following steps.
```
Comment (chri2547) [inline docs/sources/troubleshooting/_index.md:28]:
```suggestion
If you have an issue with your Dashboard panel, you can send us debug information. For more information, refer to [Send a panel to Grafana Labs support]({{< relref "./send-panel-to-grafana-support/" >}}).
```

**Comment by chri2547** [comment]
> > I kind of missed it because the Table of content did not show it inside the main Troubleshooting page as it is listed as a sub-page.
> 
> @chri2547 We could consider adding a `section` shortcode here?
> 
> I'll leave it up to you :)

I like that idea. Let me work on this PR and then I'll have both of you take a quick look.

**Comment by chri2547** [comment]
@usmangt , @jdbaldry I added a link from the troubleshooting landing page to the child topic. If looks good to you, we can merge it. Thanks!

**Review by chri2547 — APPROVED** [review]

**Comment by jdbaldry** [comment]
I believe it makes sense to backport this only as far as v9.2.x. v9.1.x has a different troubleshooting layout (https://github.com/grafana/grafana/tree/v9.1.x/docs/sources/troubleshooting).

**Comment by usmangt** [comment]
Just looked back at the original Troubleshooting page and looks good :slightly_smiling_face:  Thanks

**Pull request merged**